README.md

```diff
@@ -154,6 +154,44 @@ To track deep links in iOS you must add the following to your `AppDelegate.m` fi
   return YES;
 }
 ```
+### Native Anonymous ID 
+
+If you need to generate an `anonymousId` either natively or before the Analytics React Native package is initialized, you can send that value across the bridge and utilize it in the JavaScript layer. Segment does not generate this value for you, but does extend the functionality necessary to dispatch the value across the bridge. For reference, you can find a working example in the app and reference the code below: 
```

Comment:
Before React Native is initialized I would say, not the package. 

I would rephrase this: 
`you can send that value across the bridge and utilize it in the JavaScript layer. Segment does not generate this value for you, but does extend the functionality necessary to dispatch the value across the bridge`

to:
`you can send the anonymousId value from native code. The value has to be generated and stored by the caller. `

Reply:
i like it